blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -418,6 +474,20 @@ export default async function decorate(block) {
                         <div class="use-case-tag">Use Case 2</div>
                     </div>
                 </div>
+                <div class="main-message-wrapper">
+                    <span class="h3">Main Message</span>
+                    <span class="description">
+                        A major genAI release of the Photoshop beta app that delivers new and enhanced generative AI capabilities.
+                    </span>
+                </div>
+                <div class="channel-scope-wrapper">
```

Comment:
@mdickson-adbe Can you comment out this section "Key Channel Scope". we are planning to add this next phase 2.0